suggester/src/main/java/org/opengrok/suggest/popular/impl/chronicle/ChronicleMapAdapter.java

```diff
@@ -120,24 +120,29 @@ public void resize(final int newMapSize, final double newMapAvgKey) throws IOExc
 
         Path tempFile = Files.createTempFile("opengrok", "chronicle");
 
-        map.getAll(tempFile.toFile());
+        ChronicleMap<BytesRef, Integer> m;
 
-        String field = map.name();
+        try {
+            map.getAll(tempFile.toFile());
 
-        map.close();
+            String field = map.name();
 
-        Files.delete(f.toPath());
+            map.close();
 
-        ChronicleMap<BytesRef, Integer> m = ChronicleMap.of(BytesRef.class, Integer.class)
-                .name(field)
-                .averageKeySize(newMapAvgKey)
-                .entries(newMapSize)
-                .keyReaderAndDataAccess(BytesRefSizedReader.INSTANCE, new BytesRefDataAccess())
-                .createOrRecoverPersistedTo(f);
+            Files.delete(f.toPath());
```

Comment:
Also any chance this isn't executed when there's an exception in something above?